package.json

```diff
@@ -40,13 +46,13 @@
   ],
   "dependencies": {},
   "devDependencies": {
-    "chai": "^4.1.2",
+    "chai": "^4.2.0",
     "eslint-plugin-require-jsdoc": "^1.0.4",
-    "grunt": "^1.0.1",
-    "grunt-cli": "^1.2.0",
-    "grunt-contrib-clean": "^1.1.0",
-    "grunt-eslint": "^20.1.0",
+    "grunt": "^1.0.3",
```

Comment:
Please try to run the tests without `grunt`, we can run the tests using NPM script for this small pkg.

Reply:
Do you want to also avoid running clean and eslint through Grunt? These can be done as npm scripts too... In any case, since removal of the Grunt test dependency would require more changes beyond just updating, can this be processed as a separate issue/PR?

Reply:
Guess I had misunderstood you before. I've avoided grunt now, as well as the following changes.

- Linting: Ignore `coverage` (redundant to exclude `node_modules`); add `assert` as global
- Maintenance: Add `.editorconfig`
- Testing: Use chai/register-assert
- Travis: Add 10, 12, 14; drop 7
- npm: use non-deprecated `license` format

I don't know if you want to bump `engines` and avoid Travis for the older versions, as it'd need potentially complicated conditional logic otherwise to downgrade to older nyc, etc.